HPXMLtoOpenStudio/resources/hpxml_defaults.rb

```diff
@@ -2604,4 +2621,25 @@ def self.get_orientation_from_azimuth(azimuth)
       return HPXML::OrientationNorthwest
     end
   end
+
+  def self.get_hotwater_appliances_adj_factor(unit_type, nbeds, noccs)
+    occ_to_nbr_ratio = noccs / nbeds
+    if [HPXML::ResidentialTypeApartment, HPXML::ResidentialTypeSFA].include? unit_type
+      occ_factor = occ_to_nbr_ratio**0.51
+    elsif [HPXML::ResidentialTypeSFD].include? unit_type
+      occ_factor = occ_to_nbr_ratio**0.70
+    end
+    return occ_factor
+  end
+
+  def self.get_misc_loads_adj_factor(unit_type, nbeds, noccs, cfa)
+    if [HPXML::ResidentialTypeApartment, HPXML::ResidentialTypeSFA].include? unit_type
+      c = [-0.68, 1.09]
+    elsif [HPXML::ResidentialTypeSFD].include? unit_type
+      c = [-1.47, 1.69]
+    end
+    adj_factor = (0.5 + 0.25 * (c[0] + c[1] * noccs) / 3.0 + 0.25 * cfa / 1920.0) /
+                 (0.5 + 0.25 * nbeds / 3.0 + 0.25 * cfa / 1920.0)
+    return adj_factor
+  end
```

Comment:
Hmm. I see that you derived these equations from [here](https://github.com/NREL/resstock/blob/223d5de88caf07f1d7cfd82a4600cbdeca355d56/measures/ResStockArguments/measure.rb#L491-L497) for LULs, but is it correct the same the adjustments to [plug loads](https://github.com/NREL/resstock/blob/223d5de88caf07f1d7cfd82a4600cbdeca355d56/measures/ResStockArguments/measure.rb#L432-L440)?

Reply:
I will look into this a bit.

Reply:
For LULs, it looks like the equations can be reasonably simplified to be more consistent with the hot water and appliance equations:
- SFD: `occ_factor = occ_to_nbr_ratio**0.385`
- SFA/MF: `occ_factor = 0.68 * occ_to_nbr_ratio**0.224`

![image](https://user-images.githubusercontent.com/5861765/159765264-d439254c-dda2-4996-82f1-684865448336.png)

Reply:
Sorry, disregard my previous comment. Let me investigate more.